code-styling/markdown/README.md

```diff
@@ -0,0 +1,1155 @@
+
+  ### 3.2.2. The start of the second section
+  ```
+  
+## 3. Emphasis
+
+- Bold text MUST use the double asterisk (`**`) format.
+  
+  _Correct example_:
+
+  ```Markdown
+  **This text is printed in bold!**
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  __This text is also printed in bold...__
+  ```
+
+- Italic text MUST use the underscore (`_`) format.
+  
+  _Correct example_:
+
+  ```Markdown
+  _This text is printed in italic!_
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  *This text is also printed in italic...*
+  ```
+
+- Striketrough MUST NOT be used.
+
+- Uppercase text for emphasis SHOULD NOT be used, use emphasis constructs like **bold** or _italic_ instead.
+  The following keywords are excepted from this rule (as defined in [RFC 2119][rfc2119]):
+  > "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+  > "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+
+  _Correct example_:
+  
+  ```Markdown
+  **Note**: One SHALL NOT emphasis in uppercase.
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  NOTE: IT IS NOT RECOMMENDED TO DO THIS!
+  ```
+
+- Emphasis constructs SHOULD NOT be used to introduce a multi line named section: use headers instead.
+
+## 4. Blockquotes
+
+- Blockquotes MUST use the greater-than (`>`) character.
+
+- The greater-than character MUST be followed by a single space.
+
+  _Correct example_:
+  
+  ```Markdown
+  > This is a pretty nice blockquote :)
+  ```
+  _Incorrect example_:
+  
+  ```Markdown
+  >This blockquote is lacking some space.
+  ```
+
+- Consecutive lines within a blockquote MUST also start with the greather-than (`>`) character.
+
+  _Correct example_:
+  
+  ```Markdown
+  > I told my wife she was drawing her eyebrows too high.
+  > She looked surprised.
+  ```
+  _Incorrect example_:
+  
+  ```Markdown
+  > I tried to catch fog yesterday.
+  Mist.
+  ```
+
+- Empty lines within a blockquote MUST use the greater-than (`>`) characater.
+
+  _Correct example_:
+  
+  ```Markdown
+  > Parallel lines have so much in common.
+  >
+  > It's a shame they'll never meet.
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  > You're not completely useless...
+  
+  > You can always serve as a bad example.
+  ```
+
+## 5. Lists
+
+- Lists MUST be preceded and followed by a newline.
+
+  _Correct example_:
+  
+  ```Markdown
+  Shopping list:
+  
+  - Milk
+  - Bread
+  - Cheese
+  ```
+  
+  _Incorrect example_:
+  
+  ```Markdown
+  Shopping list:
+  - Milk
+  - Bread
+  - Cheese
+  ```
+
+- The first level of items in a list MUST NOT be indented.
+
+  _Correct example_:
+  
+  ```Markdown
+  Shopping list:
+  
+  - Milk
+  - Bread
+  - Cheese
+  ```
+
+  _Incorrect example_:
+  
+  ```Markdown
+  Shopping list:
+  
+    - Milk
+    - Bread
+    - Cheese
+  ```
+
+- Subitems of a list MUST be indented by 2 spaces and MUST be preceded and followed by a newline.
+
+  _Correct example_:
+  
+  ```Markdown
+  - First level
+  
```

Comment:
I think it should be up for debate either this white line is correct here, while i agree to the readability it just seems off for me.